packages/subgraph/src/dao/dao_v1_0_0.ts

```diff
@@ -8,7 +8,6 @@ import {
   Granted,
   Revoked,
   TrustedForwarderSet,
-  SignatureValidatorSet,
```

Comment:
Our subgraph file versioning system seems off.
We now need to remove an event from the old `dao_v1_0_0 file`.
Shouldn't we instead create an `dao_v1_4_0` file?

Reply:
I think we should create a task to figure this out and block this PR.